Header header
string name
float32 xPos
float32 yPos
bool status
